functions/__fzf_search_history.fish

```diff
@@ -1,9 +1,9 @@
 function __fzf_search_history --description "Search command history using fzf. Replace the commandline with the selected command."
     # history merge incorporates history changes from other fish sessions
-    history merge
+    builtin history merge
```

Comment:
Are you sure we don't want `command` instead of `builtin`. Here, we are using the fish provided version of history and command forces us to use that.

Reply:
My bad--sorry--you are right. History is a builtin and so we want `builtin history`. Thanks for your PR!

Reply:
Thanks! Here is a reference to the builtin docs in case anyone else comes upon this https://fishshell.com/docs/current/cmds/builtin.html